src/meta/src/model/stream.rs

```diff
@@ -119,10 +121,35 @@ pub type StreamActorWithDispatchers = (StreamActor, Vec<PbDispatcher>);
 pub type StreamActorWithUpDownstreams = (StreamActor, ActorUpstreams, Vec<PbDispatcher>);
 pub type FragmentActorDispatchers = HashMap<FragmentId, HashMap<ActorId, Vec<PbDispatcher>>>;
 
+pub type FragmentDownstreamRelation = HashMap<FragmentId, Vec<DownstreamFragmentRelation>>;
+// downstream fragment_id -> original upstream fragment_id -> new upstream fragment_id
+pub type FragmentReplaceUpstream = HashMap<FragmentId, HashMap<FragmentId, FragmentId>>;
+// upstream fragment_id -> downstream fragment_id -> upstream actor_id -> downstream actor_id
+pub type FragmentNewNoShuffle = HashMap<FragmentId, HashMap<FragmentId, HashMap<ActorId, ActorId>>>;
```

Comment:
```suggestion
pub type FragmentDownstreamRelation = HashMap<FragmentId, Vec<DownstreamFragmentRelation>>;
/// downstream fragment_id -> original upstream fragment_id -> new upstream fragment_id
pub type FragmentReplaceUpstream = HashMap<FragmentId, HashMap<FragmentId, FragmentId>>;
/// upstream fragment_id -> downstream fragment_id -> upstream actor_id -> downstream actor_id
pub type FragmentNewNoShuffle = HashMap<FragmentId, HashMap<FragmentId, HashMap<ActorId, ActorId>>>;
```
Better to add some descriptive doc string, e.g., the meaning of `FragmentNewNoShuffle` isn't straightforward at the first glance.